.github/workflows/actions/yarn-install/action.yaml

```diff
@@ -1,23 +1,43 @@
 name: Yarn Install
+inputs:
+  node-version:
+    required: false
+    default: '18'
```

Comment:
Node.js 18 is an old version that will no longer be maintained after May 2025. Would you consider requiring a version here instead of silently defaulting to an old version? I don't know much about the CI setup, but the Node.js version may be something we want to centralize and define once.

Reply:
You're right ! We should definitely plan to make our codebase node18+ compatible